packages/core/src/components/forms/numericInput.tsx

```diff
@@ -148,14 +157,19 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
      */
     private static FLOATING_POINT_NUMBER_CHARACTER_REGEX = /^[Ee0-9\+\-\.]$/;
 
-    private didPasteEventJustOccur: boolean;
     private inputElement: HTMLInputElement;
 
+    /**
+     * Updating these flags need not trigger re-renders, so don't include them in this.state.
+     */
```

Comment:
`//` comment is preferred here; this is not API documentation. (try hovering on `didPasteEventJustOccur` in VS Code and you'll see what i mean)